kijiji_repost_headless/kijiji_cmd.py

```diff
@@ -83,9 +87,14 @@ def post_ad(args):
     Post new ad
     """
     [data, imageFiles] = get_inf_details(args.inf_file)
-    api = kijiji_api.KijijiApi()
-    api.login(args.username, args.password)
-    api.post_ad_using_data(data, imageFiles)
+    attempts = 1
+    while not check_ad(args) and attempts < 5:
+        if attempts > 1:
+            print("Failed Attempt #" + str(attempts) + ", trying again.")
```

Comment:
Could you also include a print statement when it fails for the 5th time? 
Also, maybe a small sleep between attempts would be a good idea.